getting_started/topics/first-boot/distribution-files-community.adoc

```diff
@@ -1,28 +1,26 @@
 
-=== Installing Distribution Files
+[id="standalone-keycloak-install"]
+=== Installing the Server
+You can install the server on Linux or Windows. The server download ZIP file contains the scripts and binaries to run the {project_name} server.
 
-Download the {project_name} Server:
+.Procedure
 
-* *keycloak-{project_version}.[zip|tar.gz]*
+. Download *keycloak-{project_version}.[zip|tar.gz]* from  https://www.keycloak.org/downloads.html[Keycloak downloads].
 
-NOTE: This file can be downloaded from https://www.keycloak.org/downloads.html[Keycloak downloads].
+. Place the file in a directory you choose.
 
-The *keycloak-{project_version}.[zip|tar.gz]* file is the server-only distribution. It contains only the scripts and binaries to run the {project_name} server.
-
-Place the file in a directory you choose and use either the `unzip` or `tar` utility to extract it.
+. Unpack the ZIP file using the appropriate `unzip` utility, such as jar or unzip.
 
++
 .Linux/Unix
 [source,bash,subs=+attributes]
 ----
-$ unzip keycloak-{project_version}.zip
-
-or
-
 $ tar -xvzf keycloak-{project_version}.tar.gz
```

Comment:
@andymunro These lines 12-18 should be unified with the file: distribution-files-product.adoc. Please, see the previous comment